ffi/Cargo.toml

```diff
@@ -25,3 +25,4 @@ path = "../engine"
 [dev-dependencies]
 regex = "1.0.1"
 indoc = "0.3.0"
+wirefilter-ffi-ctests = {"path" = "tests/ctests"}
```

Comment:
Formatting nit: keys usually are not prefixed (they don't need to be):

```rust
wirefilter-ffi-ctests = { path = "tests/ctests" }
```

Reply:
You mean renaming the `wirefilter-ffi-ctests` crate to just `ctest` ?

Reply:
@marmeladema `"path"` don't need quotes around it

Reply:
Oh yeah sorry I meant "not quoted".

Reply:
Fixed!